purchase.adoc

```diff
@@ -1,174 +1,300 @@
-## Purchase API
+= Purchase API
 
-### Service Description
-The purchase API provides information about purchases made through iZettle.
+== Description
+
+====
+Provides information about purchases made through iZettle.
+====
+
+=== Base URL
 
-### URL
 https://purchase.izettle.com
 
-### Scopes
-The Purchase API implements the following scopes:
+=== OAuth scopes
 
 - `READ:PURCHASE`
 
-=== Get purchase history
-Will include cash register information if applicable. Purchases up to three years old are available to fetch.
+'''
+
+==== Fetch purchase history
```

Comment:
I'd think the section level should be up as same as Base URL and OAuth scopes.

Reply:
Not sure what you mean here. 
`Fetch purchase history` is a REST endpoint and parameters are specified below it.
"Fetching all purchases for a merchant." - is a use case for integrator under *Examples*